src/bindings/python/setup.cfg

```
@@ -1,15 +1,51 @@
+[tox:tox]
+envlist = py3
+
+[testenv]
+skipdist=True
+skip_install=True
+deps =
+  -rrequirements.txt
+  -rrequirements_test.txt
+setenv =
+  NGRAPH_BACKEND = {env:NGRAPH_BACKEND:"CPU"}
+  PYTHONPATH = {env:PYTHONPATH}
+  OpenVINO_DIR = {env:OpenVINO_DIR}
+passenv =
+  http_proxy
+  https_proxy
+commands=
+  {envbindir}/python setup.py bdist_wheel
+  {envbindir}/pip install --no-index --pre --find-links=dist/ openvino
+  # flake8 {posargs:src/ setup.py} --exclude=src/compatibility/openvino
+  # flake8 --ignore=D100,D101,D102,D103,D104,D105,D107,D212,W503 tests/ tests_compatibility/ --exclude=tests_compatibility/test_inference_engine # ignore lack of docs in tests
+  # mypy --config-file=tox.ini {posargs:src/} --exclude=src/compatibility/openvino
+  pytest --backend={env:NGRAPH_BACKEND} tests -v -k 'not _cuda' --ignore=tests/test_onnx/test_zoo_models.py --ignore=tests/test_utils --ignore=tests/test_inference_engine
+  pytest --backend={env:NGRAPH_BACKEND} tests_compatibility/test_ngraph -v -k 'not _cuda' --ignore=tests_compatibility/test_onnx/test_zoo_models.py
```

```suggestion
  pytest --backend={env:OV_BACKEND} tests -v -k 'not _cuda' --ignore=tests/test_onnx/test_zoo_models.py --ignore=tests/test_utils --ignore=tests/test_inference_engine
  pytest --backend={env:OV_BACKEND} tests_compatibility/test_ngraph -v -k 'not _cuda' --ignore=tests_compatibility/test_onnx/test_zoo_models.py
```
Is ignoring `tests/test_inference_engine` planned?